kratos/tests/cpp_tests/processes/test_calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -154,7 +154,8 @@ namespace Testing {
         skin_part.CreateNewElement("Element3D3N", 12, { 2,5,6 }, p_properties);
 
         // Compute the discontinuous distance function (including edge distances)
-        Flags options = CalculateDiscontinuousDistanceToSkinProcessFlags::CALCULATE_ELEMENTAL_EDGE_DISTANCES;
+        Flags options = CalculateDiscontinuousDistanceToSkinProcessFlags::CALCULATE_ELEMENTAL_EDGE_DISTANCES
+            | CalculateDiscontinuousDistanceToSkinProcessFlags::CALCULATE_ELEMENTAL_EDGE_DISTANCES_EXTRAPOLATED;
```

Comment:
This implies that we are not checking the option with edge distances but without projected values. I'd leave at least one test without projected values to check that these work separately.

Reply:
There still are a lot of tests without the second flag, is that what you mean? And I also saw, that there will be errors, if the second flag is true for a test, in which we don't expect the second flag to be true.

Reply:
Yes, what I meant is to have the old tests (without any flag), tests with the `CALCULATE_ELEMENTAL_EDGE_DISTANCES` and also tests with both  `CALCULATE_ELEMENTAL_EDGE_DISTANCES` and `CALCULATE_ELEMENTAL_EDGE_DISTANCES_EXTRAPOLATED`.

Reply:
We do have that, but it is not very obvious, I just added the flag(s) for some, and for some I didn't and then I created some extra ones as well, so there are no tests, which only check for the flags and are called that way.. would you want that?

Reply:
If it's tested I'm fine (even though it's not obvious at all).

Reply:
It really isn't, so I added an additional test. Maybe things change in the future and this way it will be more clear where an error comes from, I thought. I hope that's what you prefer too - shall I push it?

Reply:
Sure. The more tests the better.